fe1-web/src/features/social/components/ChirpCard.tsx

```diff
@@ -157,13 +153,7 @@ const ChirpCard = ({ chirp, isFirstItem, isLastItem }: IPropTypes) => {
 
   const listStyle = List.getListItemStyles(isFirstItem, isLastItem);
 
-  const actionSheetOptions: ActionSheetOption[] = [];
-  if (isSender && !chirp.isDeleted && isConnected) {
-    actionSheetOptions.push({
-      displayName: STRINGS.social_media_delete_chirp,
```

Comment:
Also regarding the strings file: If you delete a `STRINGS.x` statement, check if it is still used somewhere else and delete it from `strings.ts` if it is not. Maybe we can also add some linter rule to check for this in the future 🤔

Reply:
I like linter rules. Linters are our friends 😁 

![image](https://user-images.githubusercontent.com/7854525/223171441-4ad0a597-09a0-4a87-b55d-673e5241c864.png)